internal/gengapic/client_init_test.go

```diff
@@ -343,13 +344,16 @@ func TestClientInit(t *testing.T) {
 			serv:      servPlain,
 			parameter: proto.String("go-gapic-package=path;mypackage,transport=rest"),
 			imports: map[pbinfo.ImportSpec]bool{
-				{Path: "bytes"}: true,
-				{Path: "fmt"}:   true,
-				{Path: "google.golang.org/api/option/internaloption"}:   true,
-				{Path: "google.golang.org/grpc/metadata"}:               true,
-				{Path: "google.golang.org/protobuf/encoding/protojson"}: true,
-				{Path: "io/ioutil"}: true,
-				{Path: "net/http"}:  true,
+				{Path: "bytes"}:                        true,
+				{Path: "context"}:                      true,
+				{Path: "fmt"}:                          true,
+				{Path: "google.golang.org/api/option"}: true,
```

Comment:
nit: right column alignment weird

Reply:
I ran into this exact thing myself. I think that `gofmt` may do indentation by tabs for map literals

Reply:
yeah it's a `gofmt` decision, we are just obeying ;)